.github/workflows/notice-generation.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - run: |
           git config --global user.email "chariott-bot@eclipse.org"
           git config --global user.name "Automated Notice Generation Pipeline"
-          ./tools/notice_generation.sh
+          ./notice_generation.sh
```

Comment:
Why did you move notice generation out of tools? I believe all of the other projects have this under a tools folder.

Reply:
and similarly. Maybe it makes sense to have the 'buf.work.yaml' and 'build-all-containers.sh' in the tools directory as well.